R/DataTypes.R

```diff
@@ -77,6 +78,22 @@ odbcDataType.default <- function(info, obj, ...) {
 }
 
 #' @export
+`odbcDataType.Hive` <- function(info, obj, ...) {
+  switch_type(obj,
+    factor = "VARCHAR(255)",
```

Comment:
I thought about that, but wasn't sure how homogenous you'd like to have the type conversion for this function.  I'll make the change